app/navigators/TrusteesPartnershipNavigator.scala

```diff
@@ -35,7 +36,7 @@ class TrusteesPartnershipNavigator @Inject()(val dataCacheConnector: UserAnswers
     if(mode == CheckMode || mode == NormalMode){
       checkYourAnswers(index, journeyMode(mode), srn)
     } else {
-      if(answers.allTrusteesAfterDelete.nonEmpty && answers.allTrustees(index).isCompleted) anyMoreChanges(srn)
+      if(answers.get(IsPartnershipCompleteId(index)).getOrElse(false)) anyMoreChanges(srn)
```

Comment:
shouldn't it be isTrusteeCompleteId?

Reply:
its already used in subscription journey, creating another ticket to refactor this